test/server/cards/01_SOR/leaders/CassianAndorDedicatedToTheRebellion.spec.ts

```diff
@@ -0,0 +1,130 @@
+describe('Cassian Andor, Dedicated to the Rebellion', function() {
+    integration(function(contextRef) {
+        describe('Cassian Andor\'s leader ability', function() {
+            it('should draw a card aftering dealing 3 damage to an enemy base with an attack', function() {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['daring-raid', 'daring-raid'],
+                        deck: ['k2so#cassians-counterpart', 'open-fire'],
+                        spaceArena: ['green-squadron-awing'],
+                        leader: { card: 'cassian-andor#dedicated-to-the-rebellion', deployed: false },
+                        resources: 4,
+                    },
+                    player2: {
+                        groundArena: ['wampa'],
+                        spaceArena: ['tieln-fighter'],
+                        base: { card: 'capital-city' }
```

Comment:
If we're not adding damage or anything this can just be `base: 'capital-city'`. Is there a reason to declare it at all though? There will be a base put in by default, can be referred to by `context.p2Base`

Reply:
good to know about default, cleaned up